src-extras/Database/LSMTree/Extras/UTxO.hs

```diff
@@ -0,0 +1,77 @@
+{-# LANGUAGE DeriveAnyClass             #-}
+{-# LANGUAGE DeriveGeneric              #-}
+{-# LANGUAGE DerivingStrategies         #-}
+{-# LANGUAGE GeneralisedNewtypeDeriving #-}
+{-# LANGUAGE NamedFieldPuns             #-}
+
+module Database.LSMTree.Extras.UTxO (
+    UTxOKey (..)
+  , UTxOValue (..)
+  , UTxOBlob (..)
+  ) where
+
+import           Control.DeepSeq
+import qualified Data.ByteString as BS
+import           Data.WideWord.Word256
+import           Database.LSMTree.Extras.Orphans ()
+import           Database.LSMTree.Internal.Serialise
+import           Database.LSMTree.Internal.Serialise.Class as Class
+import           GHC.Generics
+import           System.Random
+import           Test.QuickCheck
+
+{-------------------------------------------------------------------------------
+  UTxO keys
+-------------------------------------------------------------------------------}
+
+-- | A model of a UTxO key (256-bit hash)
+newtype UTxOKey = UTxOKey Word256
```

Comment:
FWIW, the utxo-like key defined for benchmarks is **34 bytes**. There is some extra two bytes.

Reply:
Just checked and I think it should be 36: it's a 256-bit hash plus a `Word64`

Reply:
It really should be 34 :-) That's what the spec says.

(The reason it's 34 is it's 32+2, where the +2 is a 16 bit transaction output index. The assumption is that Cardano could reasonably enforce that transactions don't have more than 64k outputs, and thereby use fixed size keys. Or it could use variable sized keys using a variable sized encoding for small integers. That would then typically result in 33 byte keys.)

Anyway, 34 it is.